fsevents.js

```diff
@@ -6,7 +6,11 @@
 /* jshint node:true */
 'use strict';
 
-var Native = require('./build/Release/fse');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var Native = require(binding_path);
```

Comment:
I would think at least `path.join(__dirname, 'package.json')` rather than simple concat with forward-slash to ensure win32 compatibility. Or does `require` already handle that for us? 

Anyway I agree this seems like it could be simplified.